packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1420,6 +1395,92 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * Refreshes the meeting info permission token (it's required for joining meetings)
+   *
+   * @returns {Promise}
+   */
+  public async refreshPermissionToken(): Promise<void> {
+    if (!this.meetingInfo?.permissionToken) {
+      return;
+    }
+
+    const isStartingSpaceInstantV2Meeting =
+      this.destinationType === _CONVERSATION_URL_ &&
+      // @ts-ignore - config coming from registerPlugin
+      this.config.experimental.enableAdhocMeetings &&
+      // @ts-ignore
+      this.webex.meetings.preferredWebexSite;
+
+    const destination = isStartingSpaceInstantV2Meeting
+      ? this.meetingInfo.meetingJoinUrl
+      : this.destination;
+    const destinationType = isStartingSpaceInstantV2Meeting ? _MEETING_LINK_ : this.destinationType;
+
```

Comment:
thought: shall we add additional metrics here about that the `refresh` of the permission token has happened?

Reply:
yeah, good point, I forgot about the metrics, I thought there was a separate jira for that. I'll add them now.

Reply:
done